contrib/msggen/msggen/gen/grpc.py

```diff
@@ -235,7 +236,7 @@ def generate(self, service: Service) -> None:
             self.generate_message(message)
 
 
-class GrpcConverterGenerator:
+class GrpcConverterGenerator(IGenerator):
     def __init__(self, dest: TextIO):
         self.dest = dest
         self.logger = logging.getLogger("msggen.grpc.GrpcConversionGenerator")
```

Comment:
`GrpcConversionGenerator` -> `GrpcConverterGenerator`

Reply:
Same here, not my choice @cdecker

Reply:
What I meant here is not the class name itself, but an argument to the `getLogger`

Reply:
Yep!